packages/frontend/core/src/layouts/workspace-layout.tsx

```diff
@@ -116,10 +117,48 @@ type WorkspaceLayoutProps = {
   migration?: MigrationPoint;
 };
 
+const useSyncWorkspaceBlob = () => {
+  // temporary solution for sync blob
+
+  const [currentWorkspace] = useCurrentWorkspace();
+
+  useEffect(() => {
+    const blobEngine = getBlobEngine(currentWorkspace.blockSuiteWorkspace);
+    let stopped = false;
+    function sync() {
+      if (stopped) {
+        return;
+      }
+
+      blobEngine
+        ?.sync()
+        .catch(error => {
+          console.error('sync blob error', error);
+        })
+        .finally(() => {
+          // sync every 1 minute
+          setTimeout(() => {
```

Comment:
interval?

Reply:
If there are many large blobs, the sync may take more than a minute, and we only time it after the sync is finished.